### DISCUSSION TITLE
Change color of highlighted edge

### DISCUSSION THREAD

**Discussion by imohitmayank** [imohitmayank/jaal#25] [Q&A]
Original questions asked by @perryisbusy
"Upon hovering the mouse to any node, the edges associated with that node will be highlighted as "grey". If we want to change the highlight color to "blue" instead of "grey", how to implement that? Thank you!"
<img width="960" alt="126256625-e358748e-c52b-42e4-a429-10046c63eb84" src="https://user-images.githubusercontent.com/1883779/126533232-9fa849c0-8661-418f-b95e-f1428e61e17a.png">

**Comment by imohitmayank** [comment]
Hi @perryisbusy, yes it is possible. You can use change the `color.highlight` parameter of [edge settings](https://visjs.github.io/vis-network/docs/network/edges.html)., which is exposed by visjs and supported by Jaal. You can do this by using the `vis_opts` parameter in `plot`. For an example have a look [here](https://github.com/imohitmayank/jaal#using-visjs-settings)

**Reply by imohitmayank** [reply]
Hi @perryisbusy, if this answered your question, plz accept the answer. Thank you!

**Reply by perryisbusy** [reply]
Hello Mohit, yes! It works and also thanks for directing me to the doc.

**Reply by xuli19** [reply]
> Hello Mohit, yes! It works and also thanks for directing me to the doc.

Hi, @perryisbusy, could you show your code as an example for us. Thank you!
